storagemarket/provider_dealfilter.go

```diff
@@ -46,6 +46,9 @@ func (p *Provider) getDealFilterParams(deal *types.ProviderDealState) (*dealfilt
 		ClientDealProposal: deal.ClientDealProposal,
 		DealDataRoot:       deal.DealDataRoot,
 		Transfer:           deal.Transfer,
+		IsOffline:          deal.IsOffline,
+		SkipIPNIAnnounce:   !deal.AnnounceToIPNI,
+		RemoveUnsealedCopy: !deal.FastRetrieval,
```

Comment:
Let's do this in a separate PR as it requires a version change on the deal filter JSON object

Reply:
Sure. I will remove it with a commit. Running a devnet at the moment to check the original problem. Will push the commit after.